pkg/build/raw.go

```diff
@@ -24,13 +24,22 @@ var modifyRawImageTemplate string
 
 func (b *Builder) buildRawImage() error {
 	cmd := b.createRawImageCopyCommand()
-	err := cmd.Run()
-	if err != nil {
+	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("copying the base image %s to the output image location %s: %w",
 			b.context.ImageDefinition.Image.BaseImage, b.generateOutputImageFilename(), err)
 	}
 
-	if err = b.writeModifyScript(); err != nil {
+	if err := b.modifyRawImage(b.generateOutputImageFilename(), true, true); err != nil {
+		// modifyRawImage will wrap the error, simply return it here
+		return err
+	}
+
+	return nil
```

Comment:
We can just `return b.modifyRawImage(b.generateOutputImageFilename(), true, true)` if wrapping is not necessary.